letsencrypt/cli.py

```diff
@@ -44,6 +44,11 @@
 
 logger = logging.getLogger(__name__)
 
+# For help strings, figure out how the user ran us.
+# When invoked from letsencrypt-auto, sys.argv[0] is something like:
+# /home/user/.local/share/letsencrypt/bin/letsencrypt"
+fragment = os.path.join(".local", "share", "letsencrypt")
+cli_command = "letsencrypt-auto" if fragment in sys.argv[0] else "letsencrypt"
```

Comment:
I don't like this design for a few reasons:
1) While the path to `letsencrypt` when installed from `letsencrypt-auto` will be `~/.local/share/letsencrypt/bin/letsencrypt` in most cases, it certainly won't be in all cases meaning this won't reliably work.
2) `letsencrypt-auto` is supposed to be a temporary solution until we get real packaging which we are getting at an astonishing pace. With this in mind, why do we keep baking more le-auto stuff into letsencrypt?